lib/fog/compute/google/models/image.rb

```diff
@@ -4,40 +4,49 @@ class Google
       class Image < Fog::Model
         identity :name
 
-        attribute :id
-        attribute :kind
         attribute :archive_size_bytes, :aliases => "archiveSizeBytes"
         attribute :creation_timestamp, :aliases => "creationTimestamp"
         attribute :deprecated
         attribute :description
         attribute :disk_size_gb, :aliases => "diskSizeGb"
         attribute :family
+        attribute :guest_os_features, :aliases => "guestOsFeatures"
+        attribute :id
+        attribute :image_encryption_key, :aliases => "imageEncryptionKey"
+        attribute :kind
         attribute :licenses
+
+        # A RawDisk, e.g. -
+        # {
+        #   :source         => url_to_gcs_file,
+        #   :container_type => 'TAR',
+        #   :sha1Checksum   => ,
+        # }
+        attribute :raw_disk, :aliases => "rawDisk"
+
         attribute :self_link, :aliases => "selfLink"
+        attribute :source_disk, :aliases => "sourceDisk"
+        attribute :source_disk_encryption_key, :aliases => "sourceDiskEncryptionKey"
+        attribute :source_disk_id, :aliases => "sourceDiskId"
+        attribute :source_image, :aliases => "sourceImage"
+        attribute :source_image_encryption_key, :aliases => "sourceImageEncryptionKey"
+        attribute :source_image_id, :aliases => "sourceImageId"
         attribute :source_type, :aliases => "sourceType"
         attribute :status
 
         # This attribute is not available in the representation of an
-        # 'image' returned by the GCE servser (see GCE API). However,
+        # 'image' returned by the GCE server (see GCE API). However,
         # images are a global resource and a user can query for images
         # across projects. Therefore we try to remember which project
         # the image belongs to by tracking it in this attribute.
         attribute :project
 
-        # A RawDisk, e.g. -
-        # {
-        #   :source         => url_to_gcs_file,
-        #   :container_type => 'TAR',
-        #   :sha1Checksum   => ,
-        # }
-        attribute :raw_disk, :aliases => "rawDisk"
-
         def preferred_kernel=(_args)
```

Comment:
We can probably just delete the perferred_kernel calls since they have been deprecated for a while.

Reply:
Done!